src/popup/components/PlatformIcon.vue

```diff
@@ -34,6 +34,7 @@ export default defineComponent({
 
 .platform-icon {
   height: 44px;
+  width: 40px;
```

Comment:
The issue is not caused by wrong sizing but with the fact that the element is affect by flex shrinking. I suggest to replace the change with `flex-shrink: 0`.

Reply:
Without `width: 40px;` it still doesn't look exactly like product